<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ajouterVehiculePromenadeGuiClass</class>
 <widget class="QDialog" name="ajouterVehiculePromenadeGuiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ajouterVehiculePromenadeGui</string>
  </property>
  <widget class="QLabel" name="labelImmatriculation">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Immatriculation</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNiv">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>53</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>NIV</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leNiv">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOk">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>220</y>
     <width>80</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPoids">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre de place</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leNbPlace">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leImmatriculation">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonOk</sender>
   <signal>clicked()</signal>
   <receiver>ajouterVehiculePromenadeGuiClass</receiver>
   <slot>validerVehiculePromenade()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>validerVehiculePromenade()</slot>
 </slots>
</ui>
